main.js

```diff
@@ -0,0 +1,58 @@
+const canvas = document.getElementById("draw");
+const button = document.querySelector("h2");
+
+/* get the rendering context and its drawing functions (получаем контекст визуализации и ее функции рисования) */
+const ctx = canvas.getContext("2d");
+/* set canvas size to window size */
+canvas.width = window.innerWidth;
+canvas.height = window.innerHeight;
+/*  */
+document.body.style.overflow = "hidden";
```

Comment:
Try to avoid assigning styles in javascript. This approach used in very specific cases

Reply:
Changed this line of code in css styles.